### DISCUSSION TITLE
Will experimental.css support '~'.

### DISCUSSION THREAD

**Discussion by ahabhgk** [webpack/webpack#16492] [Q&A] [answered]
css-loader will [replace '~' with ''](https://github.com/webpack-contrib/css-loader/blob/239b9ac45066587ba425450284a93b066d7a8557/src/utils.js#L94), so `import '~normalize.css';` with css-loader this will be ok, but with experimental.css this will be a resolve error.

Will experimental.css compatible with this behave?

**Comment by alexander-akait** [comment, marked as answer]
`~` is deprecated and should not be used, just write `normalize.css/normalize.css` or just remove `~`
